index.html

```diff
@@ -1540,11 +1540,8 @@ <h3>Service properties</h3>
       <h2>Identifiers</h2>
 
       <p>
-It is necessary to learn the identifiers used within a <a>DID document</a> to
-refer to the <a>DID subject</a> and the <a>DID controller</a>, as well as any
-alternate identifiers that might also be used to reference the <a>DID subject</a>.
-The following section elaborates on the mechanisms the <a>DID document</a> data
-model provides for so doing.
+
+        A <a>DID document</a> must include identifiers referring to the <a>DID subject</a>, as well as to the <a>DID controller</a>. While the primary mechanism is to use the <a href="#did-syntax">DID Syntax</a>, the <a>DID document</a> may also include alternate identifiers to reference the <a>DID subject</a>. The following section elaborates on the mechanisms the <a>DID document</a> data model provides for so doing.
```

Comment:
Wrap to 80 characters, no indents, please. Long lines make diff-reading /very/ difficult and also complicate suggesting edits in Github.